azkaban-common/src/main/java/azkaban/version/LocalVersionSetLoader.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2020 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package azkaban.version;
+
+import com.google.common.annotations.VisibleForTesting;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+@VisibleForTesting
+public class LocalVersionSetLoader implements VersionSetLoader {
```

Comment:
Can you please add documentation for the classes and public methods?

Reply:
This is a class only meant for functional testing.

Reply:
Then please add the detail that it is for functional testing?